tests/aio-create.sh

```diff
@@ -213,6 +213,8 @@ pushd /opt/openstack-ansible
 
   elif [ "${RE_JOB_SCENARIO}" == "pike" ]; then
     git checkout "stable/pike"  # Branch checkout of Pike (Current Stable)
+    # Pin flask so it stops breaking xenial and other versions
+    echo "Flask==0.12.2" >> /opt/openstack-ansible/global-requirement-pins.txt
```

Comment:
didn't we want to remove this?

Reply:
I tried to remove this and upper-constraint in tox ini,  it breaks all other gates.
I'd like to start a new overhaul to remove all tox env vars.